### DISCUSSION TITLE
Update for V16

### DISCUSSION THREAD

**Discussion by Poper123** [gityopie/odoo-addons#206] [General]
Hi Yopi (@gityopie),
you make a great tool with google maps. I used it in V15.
Is an update planned for V16? I want to use contacts_maps.
Is it a lot of work?

Greetings from Germany
Axel

**Comment by gityopie** [comment]
Hi @Poper123 
Thank you!

Yes definitely it's a lot of work 🥲 

I have a good news and a bad news

The good news is the migration is in progress, re-writing all the javascript part using the new Odoo JS framework
This is a progress so far [demo video](https://drive.google.com/file/d/18kM1hu3eWkMaojIhvwou9e88Epbyyd60/view?usp=share_link)

The bad news is I will probably won't make them open source anymore (starting from the V16)

**Reply by Poper123** [reply]
Hello,

if the price is fair, it is ok for this nice tool.
I send you a few coffees.  Maybe it helps to develop the tool faster or I can test it for you. ;-)

All the best

Axel

**Reply by gityopie** [reply]
Thank you very much @Poper123 
It will definitely give me some more fuel

I will keep you informed

**Comment by gityopie** [comment]
Hi @Poper123 

I've sent you an invitation to access the V16 repo.
It's still a work in progress, but the Google Map view is already working.

If you have time, please have a look and don't hesitate to give a feedback

Thank you
Note: please do not share the modules 🤞🏼

**Reply by Burnaviour** [reply]
Any plan to work on openstreet map?
